api/internal/service/thegraph/alphaImpl.go

```diff
@@ -53,19 +57,19 @@ func (s *theGraphServiceAlphaImpl) GetRelationship(relationshipId string) (*v0al
 
 func (s *theGraphServiceAlphaImpl) GetRelationshipType(relType *string, ipOrgId *string) (*v0alpha.RelationshipType, error) {
 	options := s.setQueryOptions(nil)
-	queryInterface := "$relType: String, $ipOrgId: String, $first: Int, $skip: Int"
```

Comment:
I think Get API doesn't need pagination and sorting since it only returns one record